games-action/minetest/minetest-0.4.17.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit cmake-utils eutils gnome2-utils user vcs-snapshot
```

Comment:
why do you need `vcs-snapshot` and `eutils`?

Reply:
`eutils` is required for the **usex** lines. `vcs-snapshot` is no longer necessary however so I'll remove it.